cypher-shell/src/main/java/org/neo4j/shell/cli/Encryption.java

```diff
@@ -0,0 +1,20 @@
+package org.neo4j.shell.cli;
+
+import javax.annotation.Nonnull;
+
+public enum Encryption
+{
+    TRUE,
+    FALSE,
+    DEFAULT;
+
+    public static Encryption parse( @Nonnull String format) {
+        if (format.equalsIgnoreCase(TRUE.name())) {
+            return TRUE;
+        } else if (format.equalsIgnoreCase( FALSE.name() )) {
+            return FALSE;
+        } else {
+            return DEFAULT;
+        }
+    }
```

Comment:
do we want to default to `DEFAULT` if the user provides an invalid argument?
what if they provide something like `"enabled"` by mistake?
no strong opinion about this, but I personally prefer to be told when I do something like that, e.g., with 
```
        try
        {
            Encryption encryption = Encryption.valueOf( format.toUpperCase() );
        }
        catch ( IllegalArgumentException e )
        {
            throw new IllegalArgumentException(
                    format( "Invalid value '%s', valid encryption values are: %s", format, Arrays.toString( Encryption.values() ) ) )
        }
```

Reply:
That error is caught earlier already. But I will add a test.